### PR TITLE
Add Get bot info API

### PR DESCRIPTION
**Pull request by imulan**
Add Get bot info API in Messaging API update in October 2020: https://github.com/line/line-bot-sdk-java/issues/582

### CODE REVIEW

File: line-bot-model/src/main/java/com/linecorp/bot/model/response/BotInfoResponse.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
+package com.linecorp.bot.model.response;
```
Comment (imasahiro) [inline line-bot-model/src/main/java/com/linecorp/bot/model/response/BotInfoResponse.java:1]: Could you add copyright header here?

**Review by imasahiro — COMMENTED** [review]
Thanks for contributing this repo. Mostly LGTM except one nit. Could you fix code style, attach copyright to file, and sign to Contributor License Agreement ?

**Comment by imulan** [comment]
Applied. Please take another look, thanks!

**Review by imasahiro — APPROVED** [review]

**Comment by imasahiro** [comment]
Merged. Thanks for the PR @imulan !

**Pull request merged**